roslibrust/src/rosbridge/client.rs

```diff
@@ -53,7 +53,7 @@ impl ClientHandleOptions {
 /// ClientHandle is clone and multiple handles can be clone()'d from the original and passed throughout your application.
 /// ```no_run
 /// # roslibrust_codegen_macro::find_and_generate_ros_messages!(
-/// #    "../assets/ros1_common_interfaces/std_msgs"
```

Comment:
Can we `concat!` the workspace path and this path so that it's always a relative path? That should work for all versions.

Reply:
Huh not actually sure if we can do that with workspace path. No workspace path var listed here: 
https://doc.rust-lang.org/cargo/reference/environment-variables.html
Looks like issue to add that variable here:
https://github.com/rust-lang/cargo/issues/3946

I guess we could concat off of CARGO_MANIFEST_DIR and that would work in both versions. Probably better that way? I'll try that out